sys/windows/syscall_windows.go

```diff
@@ -335,6 +336,23 @@ func GetDriveType(rootPathName string) (DriveType, error) {
 	return dt, nil
 }
 
+// GetVolumeInfo returns volume informatiom at the given root path.
```

Comment:
```suggestion
// GetVolumeInfo returns volume information at the given root path.
```

Reply:
I think it would be helpful to document that this returns the file system type name.

Reply:
Did so and went ahead and just renamed the function `GetFilsystemType` since that's all we're returning from the winapi call anyway.